### ISSUE TITLE
Change Request: A section for Background on the Front Page of the character sheet 

### ISSUE THREAD

**Issue by WisdomOfWombats** [free-beer/black-sword-hack#3]
Each background also gives a special ability, and it would be nice to have a section on the front page to list that. Of course, that would probably also require an item type called "background". Alternatively, this could be covered by a Gift with the Power "Background", if that is easier to implement.

**Comment by free-beer** [comment]
Yeah, given this and your other request for custom backgrounds I think this is going to require a substantial change. Not opposed to that but it will take time. Also, custom backgrounds might invalidate my character insta-creation button which I'd be sad to see go :)
